designs/dragonphy_top/constraints/gen_constraints.py

```diff
@@ -32,19 +33,83 @@
     -waveform {{0 {0.5*main_per*time_scale}}} \\
     [get_pins iacore/clk_adc]
 
-# TX clocks
-# TODO: add generated clocks in TX
+#############
+# TX clocks #
+#############
 
-create_clock -name clk_prbsgen \\
-    -period {main_per*time_scale} \\
-    -waveform {{0 {0.5*main_per*time_scale}}} \\
-    [get_pins itx/clk_prbsgen]
+# PI outputs
```

Comment:
This is where the four PI outputs are declared as clock signals (with 0, 90, 180, and 270 degrees phase shift)